cmd/buildah/images.go

```diff
@@ -24,22 +46,35 @@ var (
 			Usage: "do not print column headings",
 		},
 		cli.BoolFlag{
-			Name:  "notruncate",
+			Name:  "no-trunc, notruncate",
```

Comment:
I think we should match the CLI of kpod and docker.   No value in the alternate spelling.